clients/vscode-hlasmplugin/package.json

```
@@ -446,7 +446,8 @@
         "hlasm.syncFileEvents": {
           "type": "boolean",
           "default": true,
-          "description": "Whether or not to send file events to hlasmplugin (File created, changed or deleted). This can be disabled for performance consideration."
+          "description": "Whether or not to send file events to hlasmplugin (File created, changed or deleted). This can be disabled for performance consideration.",
```

While this is being changed, consider using `extension` instead of `plugin`:

```suggestion
          "description": "Whether or not to send file events to HLASM LS extension (file created, changed or deleted). This can be disabled for performance consideration.",
```